src/python/yaramod_python.cpp

```diff
@@ -537,6 +538,7 @@ void addExpressionClasses(py::module& module)
 	py::class_<Expression, std::shared_ptr<Expression>>(module, "Expression")
 		.def("accept", &Expression::accept)
 		.def("get_text", &Expression::getText, py::arg("indent") = std::string{})
+		.def("get_uid", &Expression::getUid)
```

Comment:
Could we maybe change it into property that would automatically default to call of `getUid`? It transform what is rather C++ style of doing things (getter methods) into Pythonic code (getter properties).